### PR TITLE
Update sequence diagrams of EditTeam AssignMember and UnAssignMember in developer guide

### PR DESCRIPTION
**Pull request by richavm14**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -362,6 +362,9 @@ Step 1. The user executes `editteam t/1 n/Team1` to change the name of first tea
 Step 2. The `editTeam` command is created by the parser using the given team index and new name. It calls
 `Model#setTeamName`to change the name of the required team.
 
+The following sequence diagram shows how the edit team operation works:
+![EditTeamSequenceDaigram](images/EditTeamSequenceDiagram.png)
```
Comment (JonathanWiguna) [inline docs/DeveloperGuide.md:366]:
```suggestion
![EditTeamSequenceDiagram](images/EditTeamSequenceDiagram.png)
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -375,6 +378,9 @@ Step 1. The user executes `assign m/1 t/1` to assign the first person as a membe
 Step 2. The `assignMember` command is created by the parser using the given member's index (global person index) and
 team's index. It calls `Model#addPersonToTeam` to add the person to given team.
 
+The following sequence diagram shows how the assign member operation works:
+![AssignMemberSequenceDaigram](images/AssignMemberSequenceDiagram.png)
```
Comment (JonathanWiguna) [inline docs/DeveloperGuide.md:382]:
```suggestion
![AssignMemberSequenceDiagram](images/AssignMemberSequenceDiagram.png)
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -388,6 +394,9 @@ Step 1. The user executes `unassign m/1 (global person index) t/1` to unAssign t
 Step 2. The `unassignMember` command is created by the parser using the given member's index (global person index) and
 team's index. It calls `Model#removePersonFromTeam` to remove the person from given team.
 
+The following sequence diagram shows how the unassign team operation works:
+![UnAssignMemberSequenceDaigram](images/UnAssignMemberSequenceDiagram.png)
```
Comment (JonathanWiguna) [inline docs/DeveloperGuide.md:398]:
```suggestion
![UnAssignMemberSequenceDiagram](images/UnAssignMemberSequenceDiagram.png)
```

**Review by JonathanWiguna — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (richavm14) [reply docs/DeveloperGuide.md]: Have updated in recent commit!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (richavm14) [reply docs/DeveloperGuide.md]: Have updated in recent commit!

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (richavm14) [reply docs/DeveloperGuide.md]: Have updated in recent commit!

**Review by JonathanWiguna — APPROVED** [review]

**Pull request merged**